app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -90,6 +88,19 @@ fun ConversationRouterHomeBridge(
     )
 }
 
+@Composable
+@ExperimentalMaterial3Api
+@ExperimentalMaterialApi
+fun onMuteStatusChange(conversationId: ConversationId, conversationState: ConversationState): ((MutedConversationStatus) -> Unit) {
```

Comment:
this will work, but we are adding additional overhead to retrieve this hiltViewModel() not a big issue, but since we already have access to it inside ConversationSheetContent, so not sure if this changes makes it any better ? also i do not understand the intention of this return lambda ?

Reply:
@trOnk12 yes. If you tell me there's no "compose" problem with the lambda I will revert back to the older solution